src/FactSystem/ParameterManager.cc

```diff
@@ -1439,7 +1521,7 @@ void ParameterManager::_loadOfflineEditingParams(void)
     }
 
     _addMetaDataToDefaultComponent();
-    _setupDefaultComponentCategoryMap();
+    _setupDefaultComponentCategoryMap(); //TODO: check if this is really only for the default, or needs to be done for all components !!
```

Comment:
This is correct. Offline editing params will never have params from anything other than the autopilot component. Comment can be removed